### PR TITLE
Adds theming for the site banner

### PR DESCRIPTION
**Pull request by ConorOkus**
Paired with @GBKS to create a site banner that changes every time the home page is visited or when the Bitcoin logo is clicked. This allows us to add as many themes as we want so we can have a diverse range of visuals to represent Bitcoin. In the footer of the banner, you will see the author of the theme with a link to the Github issue that initiated the theme and was signed off by the community. 

We also link out to a page explaining how new themes can be added to the banner and what process to follow. 

Themes can be added very simply by adding additional objects to the themes array and changing some CSS variables essentially. Would be awesome to get some more themes for this so feel free to add some if you have an idea.

[Preview](https://deploy-preview-27--sad-borg-390916.netlify.app/)

|📷|📷|
|--|--|
|screencapture-localhost-4000-1600697435288|screencapture-localhost-4000-1600697455481|
|screencapture-localhost-4000-1600697487468|screencapture-localhost-4000-1600697478824|
|screencapture-localhost-4000-1600697502384|screencapture-localhost-4000-1600697516741|
|screencapture-localhost-4000-1600699185052|screencapture-localhost-4000-1600699197441|screencapture-localhost-4000-submit-header-design-1600699822326|screencapture-localhost-4000-submit-header-design-1600699756803

### CODE REVIEW

File: _includes/home-banner.html [hunk 1]
```diff
@@ -9,12 +9,20 @@ <h1>
       </h1>
       <p>Open-source design for bitcoin products.</p>
       <div class="home-banner-buttons-wrapper">
-        <a id="slack-link-button" href="https://bitcoindesign.slack.com/" target="blank" rel="me"> Join us on Slack</a>
+        <a id="slack-link-button" href="https://bitcoindesign.slack.com/" target="blank" rel="me">Join us on Slack</a>
```
Comment (johnsBeharry) [inline _includes/home-banner.html]: Invite link should be https://join.slack.com/t/bitcoindesign/shared_invite/zt-gytq2snl-4TEWJOTKrXRCB4YLBoDunA

**Review by johnsBeharry — COMMENTED** [review]
Slight nitpick: The outline should be required, even if its the same colour as the fill otherwise this weird jumping behaviour happens.

Sep-22-2020 00-17-09

File: _includes/home-banner.html (hunk 1 above)
Comment (ConorOkus) [reply _includes/home-banner.html]: I've heard these expire after some time?

**Comment by GBKS** [comment]
@johnsBeharry maybe we need to default values to "transparent" if they are not defined. That way the border is drawn but not visible.

**Comment by GBKS** [comment]
@johnsBeharry also, do you want to come up with a theme?

File: _includes/home-banner.html (hunk 1 above)
Comment (johnsBeharry) [reply _includes/home-banner.html]: This one doesn't have an expiration, but in fact @moneyball just made me aware of http://bitcoindesigners.org in PR https://github.com/BitcoinDesign/Guide/pull/28#issuecomment-696842741 which also doesn't expire but is also easier to remember and type out.

**Comment by GBKS** [comment]
Alright, so I think this PR is ready for review and merge action. Here is a video overview of the changes: https://www.dropbox.com/s/vh9asbzz9ok3k0q/Home%20page%20banner%20PR%20overview%20201012.mov?dl=0

**Comment by danielnordh** [comment]
Copy looks good. Only minor comments;
- We use a mix of title case and sentence case in the headings. I'd lean towards sentence case and change to 'Design bitcoin for everyone'. Since 'Bitcoin Design Guide' could be considered a name I guess that can stay as it is.
- Would remove 'helping' in second header, making it '...with the goal of improving the user experience and design ...'

**Review by johnsBeharry — APPROVED** [review]
Really great work! Just made some small tweaks you can choose to accept but nothing to hold up the merging.

I tested on a view browsers as well, works as expected in the modern browsers as well as mobile devices. We can may need to look into what considerations we will make for older browsers and devices though. Just so its clear what is supported and what isn't.

I'd like for the site to be somewhat accessible no mater, maybe even offering a downloadable version for offline consumption for people with more restrictive mobile data plans. Can be done though zip packaged releases on GitHub.

**Review by johnsBeharry — APPROVED** [review]

**Pull request merged**